client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -3,427 +3,340 @@
-                                    </div>
-                                    <oui-action-menu data-compact data-align="end"
-                                                     data-ng-if="['KIMSUFI_2015','__60_FREE','DEMO_1_G','START_1_M','START_10_M','_ASPFREE'].indexOf(hosting.offer) === -1">
-                                        <button class="oui-dropdown-option ng-binding ng-scope" type="button"
-                                                data-ng-click="setAction('change-main-domain/hosting-change-main-domain', hosting)"
-                                                data-track-on="click"
-                                                data-track-name="web::hostname::general-informations::change-main-domain"
-                                                data-track-click="action">
-                                                <span data-translate="hosting_dashboard_service_main_domain_change"></span>
-                                        </button>
-                                    </oui-action-menu>
-                                </li>
-                                <li class="oui-tile__item">
-                                    <div class="oui-tile__definition">
-                                        <strong class="oui-tile__term d-block"
-                                                data-translate="hosting_dashboard_service_web"></strong>
-                                        <span data-ng-bind="hosting.attachedDomainsCount"></span>&nbsp;
-                                        <span data-ng-bind="(hosting.attachedDomainsMaxRecommanded > 1
-                                                                ? 'hosting_dashboard_service_web_info_other'
-                                                                : 'hosting_dashboard_service_web_info_1')
-                                                            | translate: { t0: hosting.attachedDomainsMaxRecommanded }"
-                                              data-ng-if="hosting.attachedDomainsMaxRecommanded > 0 && hosting.attachedDomainsMaxRecommanded < cloudWebUnlimitedQuantity"></span>
-                                    </div>
-                                </li>
-                            </ul>
+                    <li class="oui-tile__item">
+                        <div class="oui-tile__definition">
+                            <div class="row">
+                                <div class="col-lg-6">
+                                    <strong class="oui-tile__term d-block"
+                                            data-translate="common_ipv4"></strong>
+                                    <span data-ng-bind="hosting.hostingIp"
+                                          data-ng-if="hosting.hostingIp"></span>
+                                    <span data-translate="hosting_ipv4_coming_soon"
+                                          data-ng-if="hosting.hostingIp == null"></span>
```

Comment:
Loose equality is volontary ? (Just a question)

Reply:
Nope it was there already :) i just changed the `<div>`s to `<oui-tile>`s
Ill take care of these